engine/apps/telegram/models/message.py

```diff
@@ -20,7 +20,7 @@ class TelegramMessage(models.Model):
         (ACTIONS_MESSAGE, "Actions message"),
         (LOG_MESSAGE, "Log message"),
         (FORMATTING_ERROR, "Alert can not be rendered"),
-        (PERSONAL_MESSAGE, "Alert group message with action buttons and incident log"),
+        (PERSONAL_MESSAGE, "Alert group message with action buttons and alert group log"),
```

Comment:
This line here generates a migration, but I think it won't really change anything at the DB level.